features/step_definitions/element_steps.rb

```diff
@@ -33,11 +33,19 @@
 Then('I can see the welcome message') do
   expect(@test_site.home).to have_welcome_message
 
-  expect(@test_site.home.welcome_message.text).to eq('This is the home page, there is some stuff on it')
+  expect(
```

Comment:
can we get this onto 2 lines by doing 
```
expect(blah)
  .to eq(blah)
```

As that's the preffered way of doing multilines with multichains. For other items in here also.